panoptes_aggregation/tests/reducer_tests/test_tess_reducer_column.py

```diff
@@ -53,14 +53,14 @@
     'user_id': [
         1,
         2,
-        3,
+        None,
```

Comment:
I assume these reflect the non extracts that will come from the blank extractor results?

Reply:
This is a non-logged-in user's lack of a user id, which can correspond with perfectly valid extracts otherwise.

Reply:
yeah - i'm wondering why the changed in this PR, is it linked to changes in `panoptes_aggregation/tests/extractor_tests/test_shape_extractor_column.py` ?

Reply:
This was me testing if the non-logged in users were causing the error, since I changed the tests and it still passed I figured I would keep it around so the code is explicitly testing for this case.

It is not related to the changes in the shape extractor.

Reply:
thanks for clarifying what these changes are testing.